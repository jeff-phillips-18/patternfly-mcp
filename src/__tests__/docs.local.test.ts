import * as docsLocal from '../docs.local';

describe('docsLocal', () => {
  it('should return specific properties', () => {
    expect(docsLocal).toMatchSnapshot();
  });
});

