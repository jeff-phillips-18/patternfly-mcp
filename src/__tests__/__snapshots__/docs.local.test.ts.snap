// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`docsLocal should return specific properties 1`] = `
{
  "LOCAL_DOCS": [
    "[@patternfly/react-charts](/documentation/charts/README.md)",
    "[@patternfly/react-chatbot](/documentation/chatbot/README.md)",
    "[@patternfly/react-component-groups](/documentation/component-groups/README.md)",
    "[@patternfly/react-components](/documentation/components/README.md)",
    "[@patternfly/react-guidelines](/documentation/guidelines/README.md)",
    "[@patternfly/react-resources](/documentation/resources/README.md)",
    "[@patternfly/react-setup](/documentation/setup/README.md)",
    "[@patternfly/react-troubleshooting](/documentation/troubleshooting/README.md)",
  ],
}
`;
